x-pack/plugins/ml/public/jobs/new_job_new/common/job_creator/configs/job.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export type JobId = string;
+export type BucketSpan = string;
+
+export interface Job {
```

Comment:
Suggest to move this and merge with `common/types/jobs.ts`.

Reply:
I agree that this interface would be better placed in the common location, however to avoid having this clash with `MLJob` which already exists there, I'd prefer to do this in a follow up PR.
`MLJob` can then be replaced with this interface in a refactoring PR.